app/views/nominees/OrganisationNomineeContactDetailsView.scala.html

```diff
@@ -0,0 +1,64 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.nominees.routes._
+@import models.Mode
+@import views.ViewUtils._
+@import config.FrontendAppConfig
+
+@this(
+        govukLayout: templates.GovukLayoutWrapper,
+        formHelper: FormWithCSRF,
+        errorSummary: components.errorSummary,
+        button: components.button,
+        inputText: components.inputText,
+        fieldset: components.fieldset
+)
+
+@(form: Form[_], name: String, mode: Mode)(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@govukLayout(pageTitle = Some(title(form, "organisationContactDetails.title",Some("officialsAndNominees.section")))) {
+
+ @formHelper(action = OrganisationNomineeContactDetailsController.onSubmit(mode)) {
+
+  @errorSummary(form.errors)
+
+  @fieldset("organisationContactDetails.heading", headingMessageArgs = Seq(name)) {
+
+   @inputText(
+    form,
+    id = "phoneNumber",
+    name = "phoneNumber",
+    label = "organisationContactDetails.phoneNumber.label",
+    isPageHeading = false,
+    classes = Some("govuk-input govuk-input--width-20")
+   )
+   <br>
```

Comment:
is br required?

Reply:
it's required to make it look exactly the same as the prototype (which contains the <br> there) - I can either replace it with some custom css, ignore it (so the two fields will be a little closer together), or keep it as a <br>